src/chrome/content/rules/University-of-Waterloo.xml

```diff
@@ -1,64 +1,23 @@
 <!--
-	Nonfunctional subdomains:
-
-		- cacr		("Gone")
-
-
-	Problematic subdomains:
-
-		- campaign	(refused)
-		- development	(refused)
-		- cecs		(shows info; mismatched, CN: uwaterloo.ca)
-		- hr		(redirects to ^.../$)
-		- info		(redirects to ^.../$, valid cert)
-
-
-	Partially covered subdomains:
-
-		- campaign	(→ ^)
-		- info		(→ ^)
-
-
-	Fully covered subdomains:
-
-		- (www.)
-		- cecs			(→ ^)
-		- cecspilot.cecssys
-		- crysp
-		- (www.)cs
-		- csclub
-		- development		( → ^)
-		- hr			(→ ^)
-		- (www.)math
-		- ripple
+	Invalid certificate and different HTTP/HTTPS content:
+		cacr.uwaterloo.ca
```

Comment:
Can we also keep the previous comments for problematic hosts?

Reply:
I don't think we should. The old comments are not even accurate, for example https://campaign.uwaterloo.ca isn't `refused`, and there are also over a thousand other subdomains we're not mentioning.

Reply:
Since they were in the ruleset before, I would not remove them without any comment. Of course the reason in comments can be updated.

Reply:
I've updated the top comment.